internal/operands/metrics/resources.go

```diff
@@ -42,26 +43,31 @@ var RecordRulesDescList = []RecordRulesDesc{
 		Name:        "kubevirt_ssp_operator_up_total",
 		Expr:        intstr.FromString("sum(up{pod=~'ssp-operator.*'}) OR on() vector(0)"),
 		Description: "The total number of running ssp-operator pods",
+		Type:        "Gauge",
 	},
 	{
 		Name:        "kubevirt_ssp_template_validator_up_total",
 		Expr:        intstr.FromString("sum(up{pod=~'virt-template-validator.*'}) OR on() vector(0)"),
 		Description: "The total number of running virt-template-validator pods",
+		Type:        "Gauge",
 	},
 	{
 		Name:        "kubevirt_ssp_num_of_operator_reconciling_properly",
 		Expr:        intstr.FromString("sum(ssp_operator_reconciling_properly)"),
 		Description: "The total number of ssp-operator pods reconciling with no errors",
+		Type:        "Gauge",
 	},
 	{
 		Name:        "kubevirt_ssp_rejected_vms_total",
```

Comment:
Hi, I believe we should remove the kubevirt_ssp_rejected_vms_total  and also kubevirt_ssp_total_restored_common_templates recording rules, since they are only used in 1 alert and the expression is simple and there is no need to waist resources on saving them as recording rules.